generators/bdk/java/templates/src/main/resources/static/app.html

```diff
@@ -0,0 +1,51 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>App Auth Example - Module</title>
+    <link rel="stylesheet" type="text/css" href="https://cdn.symphony.com/resources/api/v2.1/styles/symphony-external-app.css" />
```

Comment:
This is the old tempo UI CSS right?
@symphony-adnane do we want to promote UI Toolkit here?